summary_workflows/identification/nextflow.config

```diff
@@ -92,12 +96,12 @@ params  {
   otherdir = "${params.output}/other"
 
   // other parameters
-  windows = "100 50 10" // type int, several values inside string separated by spaces. The first value is used for relative PAS usage calculation.
-  genome_dir = "${params.data_dir}/ref/genomes" // Must contain a .gtf file with same substring as in challenge_ids, e.g. test.genome.mm10_test.gtf
+  windows = "100 50 25 10" // type int, several values inside string separated by spaces. The first value is used for relative PAS usage calculation.
+  genome_dir = "${params.data_dir}/genomes" // Must contain a .gtf file with same substring as in challenge_ids, e.g. test.genome.mm10_test.gtf
```

Comment:
I wouldn't want to merge this, as it breaks compatibility with the APAeval test file paths